pkg/resources/grant_privileges_to_database_role.go

```diff
@@ -0,0 +1,971 @@
+	"github.com/hashicorp/go-uuid"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+)
+
+// TODO: Imported privileges (after second account will be added)
+
+var grantPrivilegesToDatabaseRoleSchema = map[string]*schema.Schema{
+	"database_role_name": {
+		Type:             schema.TypeString,
+		Required:         true,
+		ForceNew:         true,
+		Description:      "The fully qualified name of the database role to which privileges will be granted.",
+		ValidateDiagFunc: IsValidIdentifier[sdk.DatabaseObjectIdentifier](),
+	},
+	"privileges": {
+		Type:        schema.TypeSet,
+		Optional:    true,
+		Description: "The privileges to grant on the database role.",
+		ExactlyOneOf: []string{
+			"privileges",
+			"all_privileges",
+		},
+		Elem: &schema.Schema{
+			Type:             schema.TypeString,
+			ValidateDiagFunc: isNotOwnershipGrant(),
+		},
+	},
+	"all_privileges": {
+		Type:        schema.TypeBool,
+		Optional:    true,
+		Default:     false,
+		Description: "Grant all privileges on the database role.",
+		ExactlyOneOf: []string{
+			"privileges",
+			"all_privileges",
+		},
+	},
+	"with_grant_option": {
+		Type:        schema.TypeBool,
+		Optional:    true,
+		Default:     false,
+		ForceNew:    true,
+		Description: "If specified, allows the recipient role to grant the privileges to other roles.",
+	},
+	"always_apply": {
+		Type:        schema.TypeBool,
+		Optional:    true,
+		Default:     false,
+		Description: "If true, the resource will always produce a “plan” and on “apply” it will re-grant defined privileges. It is supposed to be used only in “grant privileges on all X’s in database / schema Y” or “grant all privileges to X” scenarios to make sure that every new object in a given database / schema is granted by the account role and every new privilege is granted to the database role. Important note: this flag is not compliant with the Terraform assumptions of the config being eventually convergent (producing an empty plan).",
+	},
+	"always_apply_trigger": {
+		Type:        schema.TypeString,
+		Optional:    true,
+		Default:     "",
+		Description: "This field should not be set and its main purpose is to achieve the functionality described by always_apply field. This is value will be flipped to the opposite value on every terraform apply, thus creating a new plan that will re-apply grants.",
+	},
+	"on_database": {
+		Type:             schema.TypeString,
+		Optional:         true,
+		ForceNew:         true,
+		Description:      "The fully qualified name of the database on which privileges will be granted. If the identifier is not fully qualified (in the form of <db_name>.≤database_role_name>), the command looks for the database role in the current database for the session. All privileges are limited to the database that contains the database role, as well as other objects in the same database.",
```

Comment:
is this description right?

Reply:
Removed 2nd and 3rd sentences. Only the info about what goes into this field should be sufficient